tests/ChunkingRedisClient.Tests/WhenDoingSequentialReadsAndWrites.cs

```diff
@@ -0,0 +1,43 @@
+using ChunkingRedisClient.Client;
+using ChunkingRedisClient.Client.Deleter;
+using ChunkingRedisClient.Client.Reader;
+using ChunkingRedisClient.Client.Writer;
+using ChunkingRedisClient.Keys;
+using ChunkingRedisClient.Locking;
+using ChunkingRedisClient.Tests.TestObjects;
+using FluentAssertions;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ChunkingRedisClient.Tests
+{
+    public class WhenDoingSequentialReadsAndWrites
+    {
+        [Fact]
+        public async Task CanWriteAndReadOneItem()
+        {
+            var keyGen = new GuidKeygen();
+            var redisClient =
+                new RedisClient<Guid, Product>(
+                    new RedisClientConfig
+                    {
+                        ConnectionString = "localhost:6379"
```

Comment:
use a different host name to "localhost" so that it can be directed at any IP by hosts/DNS.

Reply:
so then you'll need to add an entry pointing to local loopback in your hosts file so you can run tests locally.

Reply:
Yeh, it's just preferable as most of the time the tests won't run on the same box that is running redis.